synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -301,11 +329,16 @@ export class SynopsysBridge {
         );
       }
     } else {
-      console.info(
+      taskLib.debug(
```

Comment:
why you're replacing console.info with taskLib.debug here, we are showing this message in pipeline